pages/api/og.tsx

```diff
@@ -92,7 +87,7 @@ export default function (req: NextRequest) {
                 fontSize: 60,
                 fontStyle: 'normal',
                 letterSpacing: '-0.025em',
-                color: `#${foreground}`,
+                color: foreground,
```

Comment:
Likewise, it's a new param, but as the value may come off the untrustworthy params, and to be consistent you may want to prepend a hash if there isn't one.  That way this system consistently handles both flavors of input.